### ISSUE TITLE
Self-supervised Monocular Multi-robot Relative Localization with Efficient Deep Neural Networks. (arXiv:2105.12797v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#7356]
Self-supervised Monocular Multi-robot Relative Localization with Efficient Deep Neural Networks. (arXiv:2105.12797v1 [cs.RO])<br>
https://ift.tt/34rmBwZ<br>
<p>Relative localization is an important ability for multiple robots to perform cooperative tasks. This paper presents a deep neural network (DNN) for monocular relative localization between multiple tiny flying robots. This approach does not require any ground-truth data from external systems or manual labeling. Our system is able to label real-world images with 3D relative positions between robots by another onboard relative estimation technology. After the training from scratch in this self-supervised way, the DNN can predict the relative positions of peer robots by purely using the monocular image. This deep-learning based visual relative localization is scalable, distributed and autonomous. Simulation shows the pipeline for synthetic image generation for multiple robots with Blender and 3D rendering, which allows for preliminary validation of the designed network. Experiments are conducted on two Crazyflie quadrotors for dataset collection with random attitude and velocity. Training and testing of the proposed network on these real-world datasets further validate the self-supervised localization effectiveness in real environment.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
